.changeset/sour-scissors-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-taxes": minor
+---
+
+replaced copied sync webhook with app-sdk one
```

Comment:
This is not a commit message, this is a public changelog. It should be written well. It also should contain entire feature scope, not last change - in this case something that app-sdk was updated, code was removed etc